library/src/main/java/com/davidstemmer/screenplay/flow/Screenplay.java

```diff
@@ -110,30 +131,30 @@ public boolean isSceneInBackstack(Scene scene) {
         return false;
     }
 
-    private Backstack trimBackstack(Backstack backstack, int numItems) {
-        Backstack.Builder builder = backstack.buildUpon();
-        for (int i = 0; i< numItems; i++) {
-            builder.pop();
-        }
-        return builder.build();
-    }
-
-    private static ArrayDeque<Scene> getLastSceneBlock(Backstack backstack) {
-        Iterator<Backstack.Entry> iterator = backstack.iterator();
-        ArrayDeque<Scene> stackedScenes = new ArrayDeque<>();
-
-        while(iterator.hasNext()) {
-            Scene scene = (Scene) iterator.next().getScreen();
-            stackedScenes.push(scene);
+    /**
+     * A "scene block" is a group of associated scenes. A scene block is derived from a source list
```

Comment:
This particular definition might make sense to put at the top of the class too, but that's probably overboard now that I think about it. I found this explanation perfectly understandable, and will have to read this class again with this understanding of the term - excellent use of javadoc!